src/plugins/action_console_dummy_plugin_1/src/dummy1.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+module.exports['dummy1'] = function (parameters) {
+  try {
+    console.log('This is a dummy plugin');
+    return {
+      status: 'Success',
+      error_log: '',
+    };
+  } catch (error) {
+    return {
+      status: 'Fail',
+      error_log: 'There was a problem',
+    };
+  }
+};
```

Comment:
Delete this one?

Reply:
I think it's a good idea to keep these two dummy plugins until we have a couple more real plugins. Because they allow us to check that all the functionality of the `registerPlugins` component is working as expected. (i.e. loading their info to database, skipping if doesn't have required info, etc.)

Reply:
Oh, did you mean delete the `.js` file?
Yeah, could do, but the idea was these should be compiled independently of the main project, and so when you download the repo you get the plugins "pre-built". (Otherwise you'd have to remember to run `yarn build_plugins` before `yarn build`, or you won't get compiled plugins in the build folder.)

Reply:
I see. Thanks!